sdk/core/Azure.Core/src/Pipeline/HttpPipeline.cs

```diff
@@ -103,6 +103,7 @@ public HttpMessage CreateMessage()
         /// <returns>The message.</returns>
         public HttpMessage CreateMessage(RequestContext context)
         {
+            Argument.AssertNotNull(context, nameof(context));
```

Comment:
Since LLC methods default the `context` to null (see [CL client](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/ConfidentialLedgerClient.cs#L89) as an example, we shouldn't throw on this condition.  Would it be possible to update the method signature to use the nullable annotation instead?

Reply:
Changed to `null`